src/unstract/adapters/vectordb/qdrant/src/qdrant.py

```diff
@@ -58,23 +56,17 @@ def get_vector_db_instance(self) -> Optional[BasePydanticVectorStore]:
                 self.config.get(VectorDbConstants.EMBEDDING_DIMENSION),
             )
             url = self.config.get(Constants.URL)
-            if (
-                self.config.get(Constants.API_KEY) is not None
-                or self.config.get(Constants.API_KEY) == ""
-            ):
-                self.client = QdrantClient(
-                    url=url, api_key=self.config.get(Constants.API_KEY)
-                )
-                vector_db = QdrantVectorStore(
-                    collection_name=self.collection_name,
-                    client=self.client,
-                    api_key=self.config.get(Constants.API_KEY),
-                )
-            else:
-                self.client = QdrantClient(url=url)
-                vector_db = QdrantVectorStore(
-                    collection_name=self.collection_name, client=self.client
-                )
+            user_api_key: Optional[str] = self.config.get(
```

Comment:
@gaya3-zipstack can you simplify what we're trying to do here? Do we need the variable `user_api_key` here?

Reply:
user_api_key is just a placholder for self.config.get(Constants.API_KEY, None). I did not want line 62 to this snippet twise. So just used a variable. No other logic.

Reply:
@gaya3-zipstack I meant that we could directly do the below and it would still logically be the same right?
```suggestion
            api_key: Optional[str] = self.config.get(Constants.API_KEY, None)
```

Reply:
Got it. Thank you. Done.